src/Features/Core/Portable/FindUsages/SourceReferenceItem.cs

```diff
@@ -1,15 +1,24 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Concurrent;
+using System.Collections.Immutable;
 
 namespace Microsoft.CodeAnalysis.FindUsages
 {
+    using ReferenceInfoMap = ImmutableDictionary<string, ImmutableArray<string>>;
```

Comment:
i like this for now.  If we ever open this stuff up though, we'll likely want to create a new type for this.

Reply:
Sure, makes sense.

---
In reply to: [222369807](https://github.com/dotnet/roslyn/pull/30155#discussion_r222369807) [](ancestors = 222369807)